action.yaml

```diff
@@ -78,7 +78,7 @@ runs:
       uses: WillAbides/setup-go-faster@v1.7.0
```

Comment:
Would also be nice to have https://github.com/WillAbides/setup-go-faster/pull/16

Reply:
Upgraded to 1.8.0.
[f58c6c7](https://github.com/dominikh/staticcheck-action/pull/8/commits/f58c6c7c981129dcd4a19763085bb612a6c89ca5)